purple-mattermost.spec.in

```diff
@@ -15,8 +15,7 @@ Source0:        https://github.com/EionRobb/purple-mattermost/archive/v%{version
 # package version as available on RHEL7/EPEL7
 BuildRequires:  json-glib-devel >= 1.0.2
 BuildRequires:  libmarkdown-devel >= 2.1.8
-BuildRequires:  libpurple-devel >= 2.8.0
-BuildRequires:  mercurial >= 2.6.2
+BuildRequires:  libpurple-devel >= 2.10.7
```

Comment:
No new API, 2.10.7 is the version on RHEL/CentOS 7 (not sure why used 2.8.0 in first place ...)  Fedora 21 has 2.10-10  (https://koji.fedoraproject.org/koji/packageinfo?packageID=3460). RHEL6 has 2.7.9 (but too old json-glib and no libmarkdown so we cannot build on it anyway).

So I think it makes sense to require minimum versions with which it has been tested.